experiment/run-library.sh

```diff
@@ -248,6 +258,32 @@ function status_data_create() {
       python -c "import sys, json; obj, pos = json.JSONDecoder().raw_decode(sys.stdin.read()); assert '_shards' in obj and  obj['_shards']['successful'] >= 1 and obj['_shards']['failed'] == 0, 'Failed to upload status data: %s' % obj"
     ###status_data.py --status-data-file $sd_file --info
 
+    # Create "results-dashboard-data" data file
+    set -x
+    status_data.py \
```

Comment:
This creates unneeded fields/structure in the JSON.

Do we have `jq` installed? Maybe use something like this:

```
$ echo '{}' | jq '.group = "Core Platform" | .product = "Red Hat Satellite" | .version = "1.2.3"'
{
  "group": "Core Platform",
  "product": "Red Hat Satellite",
  "version": "1.2.3"
}
```

Reply:
I didn't use it because I didn't know it was that easy :) Will amend it in a follow-up commit.